plugins/org.eclipse.n4js.ide/src/org/eclipse/n4js/ide/xtext/server/XBuildManager.java

```diff
@@ -125,13 +124,14 @@ public String toString() {
 	private XWorkspaceManager workspaceManager;
 
 	@Inject
-	private XTopologicalSorter topoSorter;
+	private ProjectOrderInfo.Provider projectBuildOrderProvider;
 
 	private final LinkedHashSet<URI> dirtyFiles = new LinkedHashSet<>();
 
 	private final LinkedHashSet<URI> deletedFiles = new LinkedHashSet<>();
 
-	private List<IResourceDescription.Delta> unreportedDeltas = new ArrayList<>();
+	/** Holds all deltas that */
```

Comment:
There is something missing in this comment

Reply:
tension is rising ... 😄